st2actions/st2actions/notifier/notifier.py

```diff
@@ -167,7 +167,15 @@ def _post_notify_subsection_triggers(self, liveaction=None, execution=None,
             payload['execution_id'] = execution_id
             payload['status'] = liveaction.status
             payload['start_timestamp'] = isotime.format(liveaction.start_timestamp)
-            payload['end_timestamp'] = isotime.format(liveaction.end_timestamp)
+
+            try:
+                payload['end_timestamp'] = isotime.format(liveaction.end_timestamp)
+            except AttributeError:
+                # This can be raised if liveaction.end_timestamp is None, which is caused
```

Comment:
Shouldn't we just check whether `liveaction.end_timestamp` is set instead of throwing? EAFP is very Pythonic, I know, but no `end_timestamp` is expected and predictable behavior. Anyway, I'm fine with both, mostly just wondering :)